lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -350,10 +349,33 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
+    if (this.hasApplicableAudits(category)) {
+      wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
```

Comment:
should we be setting an explicit class on the not applicable case as well? a `.lh-gauge__wrapper--notapplicable` or whatever to match `.lh-gauge__wrapper--pass`, `.lh-gauge__wrapper--average`, and `.lh-gauge__wrapper--fail`? Or are the defaults enough?

Reply:
that sounds like a good idea to me 

how would I go about adding `--notapplicable` as a new class? I'm having trouble following how the `--pass`, `--average`, and `--fail` classes are being added/modified in the DOM. all I've found so far is the template below.

```
<div class="lh-scorescale">
      <span class="lh-scorescale-range lh-scorescale-range--fail">0&ndash;49</span>
      <span class="lh-scorescale-range lh-scorescale-range--average">50&ndash;89</span>
      <span class="lh-scorescale-range lh-scorescale-range--pass">90&ndash;100</span>
  </div>
```

Reply:
at least for `.lh-gauge__wrapper--pass`, etc, it's added right here :)

`.lh-gauge__wrapper--notapplicable` could be added below, in the `else` block.

Reply:
ohh oops, i missed that

thanks, added it.